app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+
+  belongs_to :post
+  belongs_to :commenter, class_name: "User"
+  has_many :likes, as: :likeable, dependent: :destroy
+
+
+  validates :content, presence: true
+  validates :commenter, presence: true
```

Comment:
We don't need to validate commenter presence :) Our `belongs_to :commenter` already takes care of that for us